### PR TITLE
Disambiguate "processing elements"

### PR DESCRIPTION
**Pull request by briantopping**
The document generally refers to "elements" as objects traversing a stream. I initially considered simply editing "processing elements" to read "processing components", but there's a section devoted to the definition of this, so better to link them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The main goal of Reactive Streams is to govern the exchange of stream data acros
 
 It is the intention of this specification to allow the creation of many conforming implementations, which by virtue of abiding by the rules will be able to interoperate smoothly, preserving the aforementioned benefits and characteristics across the whole processing graph of a stream application.
 
-It should be noted that the precise nature of stream manipulations (transformation, splitting, merging, etc.) is not covered by this specification. Reactive Streams are only concerned with mediating the stream of data between different processing elements. In their development care has been taken to ensure that all basic ways of combining streams can be expressed.
+It should be noted that the precise nature of stream manipulations (transformation, splitting, merging, etc.) is not covered by this specification. Reactive Streams are only concerned with mediating the stream of data between different [API Components](#api-components). In their development care has been taken to ensure that all basic ways of combining streams can be expressed.
```
Comment (viktorklang) [inline README.md]: Very well-spotted, Brian!
Thanks for this change, would you mind adding yourself to the CopyrightWaivers document as per the CONTRIBUTING.md?

Thanks for your contribution!

Comment (briantopping) [inline README.md]: All set, thanks for your consideration! :)

Comment (viktorklang) [inline README.md]: :+1:

**Comment by viktorklang** [comment]
@reactive-streams/contributors Merging this documentation enhancement.

**Pull request merged**